quickstarts/k6/config.yml

```diff
@@ -0,0 +1,46 @@
+slug: k6
+
+title: k6
+
+description: |
+  ## Quickstart for k6 Load Test Monitoring
+  
+  k6 can send performance metrics to New Relic through the New Relic StatsD integration. By integrating New Relic and k6, you can visualize and correlate test result data, alert on performance data, and share advanced load testing results and their KPI's with your teams. View alongside all of your other telemetry data in New Relic for added insights.
+  Installation is as simple as running the NR StatsD integration and including NR StatsD as an output in your k6 load testing script.
+
+  This quickstart includes dashboards for the following k6 charts:
+  - Number of virtual users
+  - Max, median, and average Request Durations
+  - Rate of requests (per second)
+  - Histogram bucketing requests
+  - Data sent and data received
+  - Rate of change in the number of requests
+  - Blocked requests
+  - Response timings (90th percentile)
+  - List of all available k6 metrics
+
+summary: |
+  Send k6 performance metrics to New Relic through the New Relic StatsD integration. 
+
+level: Community
+icon: logo.png
+
+authors:
+  - Kinsey
+
+keywords:
+  - k6
+  - grafana
+  - load testing
+  - statsD
+
+documentation:  
+  - name: Set up k6 to send data to New Relic
+    url: https://k6.io/docs/results-output/real-time/new-relic/
+    description: Information on how to set up k6 to send data to New Relic, including all configuration options from k6.
+  - name: Learn more about the New Relic StatsD integration
+    url: https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/statsd-monitoring-integration-version-2/
+    description: The StatsD integration is required to send k6 data to New Relic. 
+
+dashboards:
+  - k6
```

Comment:
looks like you created a data source but aren't referring to it. You can do that like this. 

`
dataSourceIds:
 - k6
`

Reply:
Thanks for the catch, @brammerl; I added the k6 datasource reference.